### PR TITLE
Feature/sync progress

### PR DESCRIPTION
**Pull request by sonle-geekyants**
**- What I did**

update package version for [issue #759](https://github.com/atsign-foundation/apps/issues/759)

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: packages/at_onboarding_flutter/lib/screen/at_onboarding_home_screen.dart [hunk 1]
```diff
@@ -697,12 +698,23 @@ class _AtOnboardingHomeScreenState extends State<AtOnboardingHomeScreen> {
...
+
       authResponse = await _onboardingService.authenticate(atsign,
           cramSecret: secret, status: widget.onboardStatus);
+
+      atSignStatus = await _onboardingService.checkAtSignServerStatus(atsign);
+      while (atSignStatus != ServerStatus.activated) {
+        await Future.delayed(const Duration(seconds: 3));
+        atSignStatus = await _onboardingService.checkAtSignServerStatus(atsign);
+        debugPrint("currentAtSignStatus: $atSignStatus");
+      }
```
Comment (sachins-geekyants) [inline packages/at_onboarding_flutter/lib/screen/at_onboarding_home_screen.dart:719]: Can you please add a max wait time to 30 seconds ?

File: packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart [hunk 2]
```diff
@@ -192,12 +192,22 @@ class _AtOnboardingActivateScreenState
...
+
       authResponse = await _onboardingService.authenticate(atsign,
           cramSecret: secret, status: OnboardingStatus.ACTIVATE);
+
+      atSignStatus = await _onboardingService.checkAtSignServerStatus(atsign);
+      while (atSignStatus != ServerStatus.activated) {
+        await Future.delayed(const Duration(seconds: 3));
+        atSignStatus = await _onboardingService.checkAtSignServerStatus(atsign);
+        debugPrint("currentAtSignStatus: $atSignStatus");
+      }
```
Comment (sachins-geekyants) [inline packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart:212]: Can you please add a max wait time to 30 seconds?

File: packages/at_onboarding_flutter/lib/screen/at_onboarding_home_screen.dart (hunk 1 above)
Comment (sonle-geekyants) [reply packages/at_onboarding_flutter/lib/screen/at_onboarding_home_screen.dart:719]: I've fixed it.

File: packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart (hunk 2 above)
Comment (sonle-geekyants) [reply packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart:212]: I've fixed it.

**Review by sachins-geekyants — APPROVED** [review]

**Pull request merged**